src/rpc/rawtransaction_util.h

```diff
@@ -5,6 +5,11 @@
 #ifndef BITCOIN_RPC_RAWTRANSACTION_UTIL_H
 #define BITCOIN_RPC_RAWTRANSACTION_UTIL_H
 
+#include <coins.h>
```

Comment:
You should just be able to forward declare the `Coin` object and leave this include in the .cpp file if you change the coins argument into a reference.

(thanks to @ryanofsky for answering my question about this)

Reply:
Just forward declaration seems to do the job without gcc complaints

Reply:
Thanks. You should pass by reference anyway, otherwise this will make a copy of the entire map.